### DISCUSSION TITLE
Unified Docs + Explain better in https://quark.sciter.com/ how to use it.

### DISCUSSION THREAD

**Discussion by martinlombana** [c-smile/sciter-js-sdk#81] [General]
Hi!

I think it would be a good idea to explain a bit how to use QUARK. It took me while to figure it out that we must use scapp.exe in order to execute it. Or, is there an executable somewhere and I missed it? 

Also, I don't know if you have considered it already, but it would be good, maybe, to unify all the docs, examples, first steps, etc, into a single place, specially now with sciter.js being the primary sciter. I think it would ease the transition and would help sciter to be selected as a development platform. If you need help or ideas I can help. I find that now, there is a lot of information, but it is scattered in many places and sometimes it is difficult to find what you are looking for. Also some old endpoints lead to pages with plain text, which might make it look like the project is outdated (while it is the contrary!). 
(Of course I am not critizising/complaining, I know this took an enormous amount of work and I respect that. I am just presenting my view as an end-user, in order to make it better). If I can help somehow, let me know how. 

Thanks!

**Comment by c-smile** [comment]
> I think it would be a good idea to explain a bit how to use QUARK. It took me while to figure it out that we must use scapp.exe in order to execute it. Or, is there an executable somewhere and I missed it?

Quark takes a folder with HTML/CSS/script/image resources and **produces monolithic executable ready to run “as it is”**:
( from https://quark.sciter.com ).

What is not clear in this explanation? Any other wordings for this are welcome.

> but it would be good, maybe, to unify all the docs, examples, first steps, etc, into a single place,

Yes,  https://sciter.com/tutorials/ but not so sure about docs - tutorials and docs are conceptually different things.

> how to use it

What did I miss here: https://quark.sciter.com/quark-application-samples/hello-world/ ?

**Reply by martinlombana** [reply]
As I see it, it implies that "quark" is some kind of executable. And there is no indication in that page to use scapp.exe to load the folder of quark. 
It is not wrong, per se, I supose, but I think (just my opinion) that it could be clearer to explain that we should use scapp.exe to load the files and use "Quark". From an end user's perspective. There is no mention of that, albeit simple, procedure in that page.

**Reply by c-smile** [reply]
From https://quark.sciter.com/quark-application-samples/hello-world/ 

> 3. Run Quark from Sciter SDK
> While Quark can be compiled by itself as a, well, Quark application, we will use the Quark in it’s “primordial” form:
> 
> Run on Windows:
> 
> cd {sciter sdk}/bin.quark/windows/
> quark-start.bat

Any other wordings are welcome.

**Reply by martinlombana** [reply]
That would suffice. I responded at the same time!

**Reply by martinlombana** [reply]
I somehow missed that. It was right all along. Sorry. I don't know but I didn't click in each of the examples, because I didn't see the link to it, they didn't seem "clickable" at first glance, and, in mobile, I didn't go further. It thought it ended there, and it was just showcasing examples that we could find elsewhere. Maybe a [read more], would suffice in that area, as a bonus to access each example. Just in case.
Sorry!
